src/apps/dashboard/routes/users/index.tsx

```diff
@@ -49,12 +49,18 @@ const UserProfiles: FunctionComponent = () => {
         const showUserMenu = (elem: HTMLElement) => {
             const card = dom.parentWithClass(elem, 'card');
             const userId = card?.getAttribute('data-userid');
+            const username = card?.getAttribute('data-username');
 
             if (!userId) {
                 console.error('Unexpected null user id');
                 return;
             }
 
+            if (!username) {
```

Comment:
Do we need to return if no username exists? if we only need the userId I think this check isn't necessary and could prevent the deletion of some buggy user with a missing username.